src/components/study-manager.js

```diff
@@ -494,7 +499,25 @@ const StudyManager = ({ onClick }) => {
         const ws = connectNotificationsWsUpdateStudies();
 
         ws.onmessage = function (event) {
-            dispatchStudies();
+            let eventData = JSON.parse(event.data);
+            if (eventData.headers) {
+                const error = eventData.headers['error'];
+                if (error !== undefined) {
+                    const studyName = eventData.headers['studyName'];
+                    const errorMessage =
```

Comment:
Perhaps a const for hearder name `error` or const enum for all header names of te web socket (study)